vnext/ReactUWP/Views/ShadowNodeBase.cpp

```diff
@@ -99,12 +100,18 @@ winrt::Windows::UI::Composition::CompositionPropertySet ShadowNodeBase::EnsureTr
 // "center":  This is the center of the UIElement
 // "transform": This will hold the un-centered TransformMatrix we want to apply
 void ShadowNodeBase::UpdateTransformPS() {
+  auto uielement = m_view.try_as<winrt::UIElement>();
+  assert(uielement != nullptr);
+
+  if (g_HasActualSizeProperty == TriBit::Undefined) {
+    // ActualSize works on 19H1+ only
+    g_HasActualSizeProperty = (uielement.try_as<winrt::IUIElement10>()) ? TriBit::Set : TriBit::NotSet;
+  }
```

Comment:
I suggest moving this logic to as early as possible - how about doing this in createView()?  I'm not worried about the perf cost of doing this one-time query.

Reply:
yeah makes sense, moved over : )